src/main/views/status-description.njk

```diff
@@ -0,0 +1,68 @@
+{% from "govuk/components/table/macro.njk" import govukTable as 'govukTable' %}
+{% from "./macros/common-components.njk" import goBack %}
+{{ displayAlpha }}
+{% extends "template.njk" %}
+{% block pageTitle %}
+  {{ "Court Event Status" | safe }}
+{% endblock %}
+{% block beforeContent %}
+  {{ goBack('/') }}
+{% endblock %}
+{% block content %}
+  <div id="grid-row" class="govuk-grid-row">
+    <div class="layout-width-four-fifths">
+      <h2 id='page-heading' class="govuk-heading-l">Live hearing updates - glossary of terms</h2>
+      <p class="govuk-body">A list of descriptions of each hearing status that reflects in-court events</p>
```

Comment:
I think this should be 'A list of descriptions for each' based on the screenshot

Reply:
Thanks Chris. I have fixed the typo.